src/main/resources/default-recheck.ignore

```diff
@@ -3,10 +3,10 @@
 # You can ignore specific attributes like so: 
 # matcher: retestid=div-b4f23, attribute: font.*
 
-# To ignore whole subtrees, use e.g. xpath:
-# matcher: xpath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
+# To ignore whole subtrees, use e.g. XPath:
+# matcher: XPath=HTML[1]/BODY[1]/DIV[1]/DIV[1]/DIV[1]
```

Comment:
While changing the actual comment is fine, you must not change the line `match: …`. `ElementXPathMatcher` requires `xpath`.